### PR TITLE
Dell Emc collections migrations

### PR DESCRIPTION
**Pull request by javeedf**
request to move the Dell EMC networking related files to ansible_collection

### CODE REVIEW

File: scenarios/nwo/dellemc_networking.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+os10:
```
Comment (anshulbehl) [inline scenarios/nwo/dellemc_networking.yml]: Please sort numerically [os6,os9,os10]]

File: scenarios/nwo/dellemc_networking.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  action:
+  - dellos10.py
+  cliconf:
+  - dellos10.py
+  doc_fragments:
+  - dellos10.py
+  integration:
+  - dellos10_*/*
```
Comment (anshulbehl) [inline scenarios/nwo/dellemc_networking.yml]: Please specify each file instead of wildcards

File: scenarios/nwo/dellemc_networking.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  - dellos10.py
+os9:
+  action:
+  - dellos9.py
+  cliconf:
+  - dellos9.py
+  doc_fragments:
+  - dellos9.py
+  integration:
+  - dellos9_*/*
```
Comment (anshulbehl) [inline scenarios/nwo/dellemc_networking.yml]: specify file instead of wildcard

File: scenarios/nwo/dellemc_networking.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  - dellos9.py
+os6:
+  action:
+  - dellos6.py
+  cliconf:
+  - dellos6.py
+  doc_fragments:
+  - dellos6.py
+  integration:
+  - dellos6_*/*
```
Comment (anshulbehl) [inline scenarios/nwo/dellemc_networking.yml]: specify each file instead of wildcard

File: scenarios/nwo/dellemc_networking.yml (hunk 1 above)
Comment (anshulbehl) [inline scenarios/nwo/dellemc_networking.yml]: the file name is not a namespace yet in galaxy and collections don't exist at the moment, please create the namespace first

File: scenarios/nwo/dellemc_networking.yml (hunk 2 above)
Comment (gundalow) [reply scenarios/nwo/dellemc_networking.yml]: Wildcards are OK here

Comment (anshulbehl) [reply scenarios/nwo/dellemc_networking.yml]: ack, I wasn't aware

File: scenarios/nwo/dellemc_networking.yml (hunk 1 above)
Comment (gundalow) [reply scenarios/nwo/dellemc_networking.yml]: As long as namespace is created before the first publish to Galaxy we should be OK.

Comment (anshulbehl) [reply scenarios/nwo/dellemc_networking.yml]: ack

**Pull request merged**